public/app/ui/components/collections/library/detail.html

```diff
@@ -1,120 +1,114 @@
-<section class="detail detail-library">
-    <div class="row">
-        <div class="col-md-12">
-            <div class="pull-right" data-bind="component: displayFor('MediaMedium')"></div>
-
+<table class="table table-condensed library-detail-table">
```

Comment:
The changes here are probably clearer with `?w=1`.  Essentially the `MediaMedium` has come out (see `pages/detail/detail.html`) and everything else is in a single table (previously it was two tables to simulate two columns).